### ISSUE TITLE
Background opacity is not working

### ISSUE THREAD

**Issue by Sim923** [ajsmth/tailwind-react-native#4]
Thank you for this wonderful library!

But I was unable to implement background-opacity.
https://tailwindcss.com/docs/background-opacity

So I created my own customized styles.

I would be grateful it could support bg-opacity.
I reviewed this library - https://github.com/vadimdemedes/tailwind-rn
And it supports bg-opacity fully!

**Comment by Sim923** [comment]
Sure,
I did it exactly and got the desired result

But I expected to use the same tailwind attributes just like bg-opacity-50

**Comment by ajsmth** [comment]
yes - they are leveraging css variables which are not available in RN - I know the tailwind-rn package does some JS stuff to emulate css variables which might be worthwhile in the future, but for now this is the best solution

**Comment by Sim923** [comment]
Thank you!
